src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Does it make sense to add unit tests to this class?

Reply:
This class is tightened to the CDP, which makes it extremely hard to unit test. We can rely on e2e tests.